### DISCUSSION TITLE
How to get latest version from source

### DISCUSSION THREAD

**Discussion by elgarbe** [abraunegg/onedrive#2250] [Q&A] [answered]
Hi, I've compiled and install a PR from source. Now I see that there is a new version available. What is the correct steps to get, build and install latest from source?
Thank

**Comment by abraunegg** [comment, marked as answer]
@elgarbe 
> Hi, I've compiled and install a PR from source. Now I see that there is a new version available. What is the correct steps to get, build and install latest from source?

Given that you were previously using the Ubuntu packages from the OpenSuSE Build Service repository, it is best that you go back to using that repository for the package for your platform.

So the right method here would be:
* Uninstall the manually built client: https://github.com/abraunegg/onedrive/blob/master/docs/INSTALL.md#uninstalling-the-client-if-installed-and-built-from-source
* Install the client from the OpenSuSE Build Service: https://github.com/abraunegg/onedrive/blob/master/docs/ubuntu-package-install.md
